root/static/scripts/place.js

```diff
@@ -10,7 +10,7 @@ const _ = require('lodash');
 
 const isBlank = require('./common/utility/isBlank');
 const initializeDuplicateChecker = require('./edit/check-duplicates');
```

Comment:
```suggestion
const initializeDuplicateChecker = require('./edit/check-duplicates').default;
```
There's a similar issue in root/static/scripts/series.js